docs/environment.yaml

```diff
@@ -6,7 +6,9 @@ channels:
   - anaconda
 
 dependencies:
-  - python==3.7
+  - python=3.7
+  - sphinx=2.2.1
+  - sphinx_rtd_theme
   - pip
   - pip:
-    - sphinxcontrib-programoutput
```

Comment:
Do we even need `sphinxcontrib-programoutput` all together now? Since we are not outputing the result directly I think we can delete. What do you say?

Reply:
Update on this